helix-term/src/commands.rs

```diff
@@ -2085,269 +2085,266 @@ mod cmd {
             completer: None,
         },
         TypableCommand {
             name: "primary-clipboard-paste-after",
-            alias: None,
+            aliases: &[],
             doc: "Paste primary clipboard after selections.",
             fun: paste_primary_clipboard_after,
             completer: None,
         },
         TypableCommand {
             name: "primary-clipboard-paste-before",
-            alias: None,
+            aliases: &[],
             doc: "Paste primary clipboard before selections.",
             fun: paste_primary_clipboard_before,
             completer: None,
         },
         TypableCommand {
             name: "primary-clipboard-paste-replace",
-            alias: None,
+            aliases: &[],
             doc: "Replace selections with content of system primary clipboard.",
             fun: replace_selections_with_primary_clipboard,
             completer: None,
         },
         TypableCommand {
             name: "show-clipboard-provider",
-            alias: None,
+            aliases: &[],
             doc: "Show clipboard provider name in status bar.",
             fun: show_clipboard_provider,
             completer: None,
         },
         TypableCommand {
             name: "change-current-directory",
-            alias: Some("cd"),
+            aliases: &["cd"],
             doc: "Change the current working directory (:cd <dir>).",
             fun: change_current_directory,
             completer: Some(completers::directory),
         },
         TypableCommand {
             name: "show-directory",
-            alias: Some("pwd"),
+            aliases: &["pwd"],
             doc: "Show the current working directory.",
             fun: show_current_directory,
             completer: None,
         },
         TypableCommand {
             name: "encoding",
-            alias: None,
+            aliases: &[],
             doc: "Set encoding based on `https://encoding.spec.whatwg.org`",
             fun: set_encoding,
             completer: None,
         },
         TypableCommand {
             name: "reload",
-            alias: None,
+            aliases: &[],
             doc: "Discard changes and reload from the source file.",
             fun: reload,
             completer: None,
         },
         TypableCommand {
             name: "tree-sitter-scopes",
-            alias: None,
+            aliases: &[],
             doc: "Display tree sitter scopes, primarily for theming and development.",
             fun: tree_sitter_scopes,
             completer: None,
         },
         TypableCommand {
             name: "vsplit",
-            alias: Some("vs"),
+            aliases: &["vs"],
             doc: "Open the file in a vertical split.",
             fun: vsplit,
             completer: Some(completers::filename),
         },
         TypableCommand {
             name: "hsplit",
-            alias: Some("sp"),
+            aliases: &["hs", "sp"],
```

Comment:
Where does this `hs` come from`?

Reply:
For horizontal-split, it's not used by others editor I think but it looked quite logical to me if we have the `vs`